presto-hive-hadoop2/pom.xml

```diff
@@ -65,6 +65,43 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- For ADLS Gen2 in HDInsight -->
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
+            <artifactId>jackson-mapper-asl</artifactId>
+            <version>1.9.13</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.squareup.okhttp</groupId>
+            <artifactId>okhttp</artifactId>
```

Comment:
We should update the Hadoop shading to include this (and `okio`) rather than adding it here